diff --git a/node_modules/@whiskeysockets/baileys/lib/Utils/validate-connection.js b/node_modules/@whiskeysockets/baileys/lib/Utils/validate-connection.js
index 6a65a73..01cb9cf 100644
--- a/node_modules/@whiskeysockets/baileys/lib/Utils/validate-connection.js
+++ b/node_modules/@whiskeysockets/baileys/lib/Utils/validate-connection.js
@@ -15,7 +15,7 @@ const getUserAgent = (config) => {
         },
         platform: config.browser[1].toLocaleLowerCase().includes('android')
             ? proto.ClientPayload.UserAgent.Platform.ANDROID
-            : proto.ClientPayload.UserAgent.Platform.WEB,
+            : proto.ClientPayload.UserAgent.Platform.MACOS,
         releaseChannel: proto.ClientPayload.UserAgent.ReleaseChannel.RELEASE,
         osVersion: '0.1',
         device: 'Desktop',
